chapters/functions.tex

```diff
@@ -2046,10 +2046,9 @@ \subsubsection{Simple Types}\label{simple-types}
 \end{tabular}
 \end{center}
 
-Sending string literals to FORTRAN~77 subroutines/functions is supported
-for Lapack/Blas-routines, and the strings are \textsc{nul}-terminated for
-compatibility with C. Returning strings from FORTRAN~77
-subroutines/functions is currently not supported.
+Sending string literals to FORTRAN~77 subroutines/functions is supported for Lapack/Blas-routines, and the strings are \textsc{nul}-terminated for compatibility with C.
```

Comment:
Could a FORTRAN expert shed some light on this?  Why would we need to define null-termination of strings passed to FORTRAN?

Reply:
Since the NUL-termination wasn't changed I don't see it as relevant for this PR.
If the Fortran-code is transformed into C-code using f2c I can see that it helps; and if the strings already are C-strings there's zero cost.

Reply:
Right, it isn't relevant for this PR.  I see that there is probably zero cost for all tools, but I wouldn't have expected this to be of any value even when using f2c.  Maybe you can elaborate on this some other time…

Resolving.